src/uitpas/search/detail/UiTPasAdvantageDescription.js

```diff
@@ -167,7 +167,11 @@ export default class UiTPasAdvantageDescription extends React.Component {
   }
 
   onlineExchangeUrl() {
-    return `${process.env.REACT_APP_UITPAS_URL}/omruilen/${this.props.advantage.id}`;
+    const hostName = document.location.hostname;
+    const baseUrl = hostName.startsWith("test")
+      ? process.env.REACT_APP_UITPAS_TEST_URL
+      : process.env.REACT_APP_UITPAS_PROD_URL;
+    return `${baseUrl}/omruilen/${this.props.advantage.id}`;
```

Comment:
Why do we need to differentiate between test and production URLs here? Will there ever be a case where we will get test advantages on production or vice-versa?

And why can we not use the existing `PUBLIC_URL` env variable?

Reply:
The reason why we need the TEST url is because we need to test the changes on test.uitpas.be otherwise you will be redirected to production uitpas.be but the new code isn't available on prodcution yet. 

The PUBLIC_URL is the url on which the site is hosted. 
In the case of uitpas.be this could make sense. However this module is also used on https://uitinhetmeetjesland.be/content/voordelen-uitpas?cardSystemsFilter[0]=UiTPAS%20Meetjesland#/voordelen
If the user clicks on the "online exchange button" the user should be redirected to uitpas.be which is another url than the PUBLIC_URL.

Reply:
Okay, but why not just have a `REACT_APP_UITPAS_URL` then and set that to the right URL on the test/production environment respectively?

Reply:
The file it's in now is called `.env.production`, so you're setting an env for the test environment in the production config no?

Reply:
The problem is that we currently need to build the application and copy paste the build files into the uitpas application. So the version on test and prod are using the same value because it is baked into the build:
See: https://github.com/cultuurnet/uitpasbe-frontend/pull/18/commits/7b330fc5d71d8174882688b5c06c0505d6360178

Reply:
Hmm I see. But that seems less than ideal in general no? Is there no way to improve that? Maybe something that Geoffrey can give some advice on?

Reply:
@simon-debruijn had a good idea.
See latest commit.

Reply:
Nice solution @simon-debruijn ! 👏